### PR TITLE
refactor: uncomment some tests, update `$dt$epoch()`

### PR DESCRIPTION
**Pull request by etiennebacher**

### CODE REVIEW

File: man/DataFrame_class.Rd [hunk 1]
```diff
@@ -116,9 +116,7 @@ withr::with_timezone(
   \}
 )
 #> <error: in to_r: ComputeError(ErrString("datetime '2020-03-08 02:00:00' is non-existent in time zone 'America/New_York'. You may be able to use `non_existent='null'` to return `null` in this case.")) When calling: devtools::document()>
-}\if{html}{\out{</div>}}
```
Comment (eitsupi) [inline man/DataFrame_class.Rd:119]: I think they are unrelated changes.
@etiennebacher Maybe you haven't updated R packages? (maybe knitr)

Comment (etiennebacher) [reply man/DataFrame_class.Rd:119]: I have everything up to date. Those were added in #1195, I guess by mistake

Comment (eitsupi) [reply man/DataFrame_class.Rd:119]: Hmmm, how about the pandoc version?
